libraries/botbuilder/src/teamsActivityHandler.ts

```diff
@@ -44,43 +44,43 @@ export class TeamsActivityHandler extends ActivityHandler {
             } else {
                 switch (context.activity.name) {
                     case 'fileConsent/invoke':
-                        return TeamsActivityHandler.createInvokeResponse(await this.handleTeamsFileConsent(context, context.activity.value));
+                        return ActivityHandler.createInvokeResponse(await this.handleTeamsFileConsent(context, context.activity.value));
```

Comment:
I don't quite understand these changes. Is the old code a bug?

Reply:
He just moved it to the base class because he needs it down there now

---
In reply to: [417531893](https://github.com/microsoft/botbuilder-js/pull/2137#discussion_r417531893) [](ancestors = 417531893)

Reply:
createInvokeResponse was moved to the ActivityHandler in this PR, but @johnataylor should it have been moved to ActivityHandlerBase?

Reply:
no i moved a helper function from TeamsActivityHandler up into the more generic ActivityHandler - it used to be Invokes (and therefore this helper) were all Teams specific.

Reply:
we don't have much logic in ActivityHandlerBase at all - we use it in ActivityHandler (well now we do) so I just moved it that far.